mycroft/util/parse.py

```diff
@@ -214,6 +205,10 @@ def extract_datetime(text, anchorDate=None, lang=None, default_time=None):
     common ways that humans express dates and times, including relative dates
     like "5 days from today", "tomorrow', and "Tuesday".
 
+    The "next" instance of a day or weekend is considered to be no earlier than
+    48 hours in the future. On Friday, "next Monday" would be in 3 days.
+    On Saturday, "next Monday" would be in 9 days.
+
```

Comment:
Should this be added to the general functions description / specification. This is as far as I can tell something that's different in different cultures / languages and might be better left unspecified in the general function